src/Commands/AdminCommand.php

```diff
@@ -117,12 +122,19 @@ protected function getUser($create = false)
         if ($create) {
             $name = $this->ask('Enter the admin name');
             $password = $this->secret('Enter admin password');
+            $confirmPassword = $this->secret('Confirm Password'); 
 
             // Ask for email if there wasnt set one
             if (!$email) {
                 $email = $this->ask('Enter the admin email');
             }
 
+            // passwords donot match
+            if( $password != $confirmPassword){
+                $this->info('Passwords donot match');
```

Comment:
Please change this to something like `Passwords don't match`

Reply:
updated the comment.